src/extension.ts

```diff
@@ -69,7 +69,7 @@ export function activate(context: vscode.ExtensionContext) {
   });
   context.subscriptions.push(disposableOneJsontracer);
 
-  context.subscriptions.push(MondrianEditorProvider.register(context));
+  MondrianEditorProvider.register(context);
```

Comment:
From clean code's view, a little bit of improvement is that it does registration at one place. (not passing the provider disposable to the caller)